app/queries/appeals_updated_since_query.rb

```
@@ -9,11 +9,7 @@ def initialize(since_date:)
   end
 
   def call
-    # query can take up to 90 seconds
-    ActiveRecord::Base.connection.execute "SET statement_timeout = 90000"
```

`build_query` does not actually execute the query.

And, `ActiveRecord::Base` defaults to the wrong db. We want to use the ETL db explicitly.